lib/breakers/uptime_middleware.rb

```diff
@@ -49,7 +49,9 @@ def outage_response(outage:, service:)
     end
 
     def handle_request(service:, request_env:, current_outage: nil)
+      start_time = Time.now
       return @app.call(request_env).on_complete do |response_env|
+        response_env[:duration] = Time.now - start_time
```

Comment:
This stackoverflow might be insightful as well: http://stackoverflow.com/questions/3561381/custom-http-headers-naming-conventions